pkgs/tools/misc/qarma-for-qt6/default.qmake.nix

```diff
@@ -0,0 +1,139 @@
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, wrapQtAppsHook
+, qtbase
+, qmake
+, qt5compat
+, pkg-config
+, libglvnd, libxkbcommon, vulkan-headers # TODO should be inherited from qtbase
+}:
+
+/*
+FIXME include paths should be $dev not $out
+build flags: -j4 -l4 SHELL=/nix/store/wadmyilr414n7bimxysbny876i2vlm5r-bash-5.1-p8/bin/bash
+g++ -c -pipe -O2 -std=gnu++1z -Wall -Wextra -D_REENTRANT -fPIC -DWS_X11 -DQT_NO_DEBUG
+ -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_DBUS_LIB -DQT_CORE_LIB -I.
+ -I/nix/store/4frx7964wbnsq76ybsfphqr1b9vr71kn-qtbase-6.2.1/include
+ -I/nix/store/4frx7964wbnsq76ybsfphqr1b9vr71kn-qtbase-6.2.1/include/QtWidgets
+
+user@laptop1:/nix/store/4gllgq492npdmvwc6gb5kvi119libj21-qtbase-6.2.1-dev/mkspecs
+./modules/qt_lib_widgets.pri:6:QT.widgets.includes = $$QT_MODULE_INCLUDE_BASE $$QT_MODULE_INCLUDE_BASE/QtWidgets
+
+/nix/store/4gllgq492npdmvwc6gb5kvi119libj21-qtbase-6.2.1-dev/mkspecs/features/qt_config.prf
+         QT_MODULE_INCLUDE_BASE = $$[QT_INSTALL_HEADERS]
+         QT_MODULE_LIB_BASE = $$[QT_INSTALL_LIBS]
+         QT_MODULE_HOST_LIB_BASE = $$[QT_HOST_LIBS]
+         QT_MODULE_HOST_LIBEXEC_BASE = $$[QT_HOST_LIBEXECS]
+         QT_MODULE_BIN_BASE = $$[QT_INSTALL_BINS]
+
+
+*/
+
+mkDerivation rec {
+  pname = "qarma";
+  version = "2021-10-05";
+
+  src = fetchFromGitHub {
+    name = "${pname}-${version}-source";
+    owner = "luebking";
+    repo = pname;
+    rev = "605ea4213406718ba869dd146875195e57488786";
+    sha256 = "KFoFywFeGqNmE1y49DrXJZ1jIK5jMOCOspkkFME+DR8=";
+  };
+
+  # FIXME only needed for qt6
+  # set attributes for qt-6/hooks/qmake-hook.sh
+  # TODO better. we do not want to set this for every libsForQt6.callPackage target
+  inherit (qtbase) qtDocPrefix qtQmlPrefix qtPluginPrefix;
+
+  patches = [
+    ./qarma-qt6.patch
+  ];
+
+  patchPhase = ''
```

Comment:
Currently the above patch is not applied.

Reply:
fixed in 591178c668a28882b402538d4365b7342b751b7b